### ISSUE TITLE
Create Section: User Authentication via Workforce IdP

### ISSUE THREAD

**Issue by jischr** [openid/ipsie#19]
topic should cover
- set up user authentication via federated relationship with a customer's workforce IdP

**Comment by joninusa** [comment]
This could include sharing MFA timeout or retry settings.e.g. For automated authentications, when MFA is required, don't exceed the retries to prevent account lockouts

**Comment by joninusa** [comment]
Add Location sharing which can include the real client IP address

**Comment by dhs-BI** [comment]
@joninusa can you describe more about "automated authentications" and the use cases that would require them?

**Comment by joninusa** [comment]
> @joninusa can you describe more about "automated authentications" and the use cases that would require them?

I updated my original comment, does this answer your question?

**Comment by dhs-BI** [comment]
Yes, that helps. My perspective is that the user should be involved in the authentication ceremony, e.g. NIST S800-63B and the concept of activation secrets for multifactor cryptographic authenticators.  However, we should collect the relevant use cases for silent/automated authentication of the user and determine whether they are in/out of scope for IPSIE.
